mitosheet/src/mito/components/icons/SuggestedStyle3Icon.tsx

```diff
@@ -5,7 +5,7 @@ import React from 'react';
 
 const SuggestedStyle3Icon = (): JSX.Element => {
     return (
-        <svg width="58" height="46" viewBox="0 0 58 46" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <svg width="50" height="46" viewBox="0 0 58 46" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I'm fine leaving this, but just calling out that this looks worse in Jupyter as a result. 

Now that we're supporting different size task panes, we should make sure to think about that in our Figma designs! 

<img width="505" alt="Screenshot 2023-07-18 at 11 25 50 AM" src="https://github.com/mito-ds/monorepo/assets/18709905/e5a40094-c3dc-403b-b507-a51de82c8950">